Makefile

```diff
@@ -333,7 +334,9 @@ build-package:
 	@chmod 777 ${PWD}/hack/hypershift/package/hcp/addon-operator.yaml
 	$(CONTAINER_ENGINE) run --privileged --rm -v ${PWD}:/workdir quay.io/app-sre/yq:4 -i '.spec.template.spec.containers[0].image = "$(OPERATOR_IMAGE_URI)"' \
 	hack/hypershift/package/hcp/addon-operator.yaml
-	$(CONTAINER_ENGINE) build -t $(PKG_IMG):$(PKG_IMAGETAG) -f $(join $(CURDIR),/hack/hypershift/package/addon-operator-package.Containerfile) . && \
+	$(CONTAINER_ENGINE) run --rm -v $(CURDIR):/hack:z $(PKO_CLI_IMAGE) update ./hack/hypershift/package
+	$(CONTAINER_ENGINE) run --rm -v $(CURDIR):/hack:z $(PKO_CLI_IMAGE) build ./hack/hypershift/package -t $(PKG_IMG):$(IMAGETAG) -o /hack/hypershift/package/validating-webhooks-package.oci
+	$(CONTAINER_ENGINE) import $(join $(CURDIR),/hack/hypershift/package/validating-webhooks-package.oci) $(PKG_IMG):$(IMAGETAG)
```

Comment:
This refers to validating webhooks package, a copy-paste error.

Reply:
okay, let me take a look.